openstates/utils/lxmlize.py

```diff
@@ -1,12 +1,29 @@
 import requests
 import lxml.html
+import subprocess
 
 
 def url_xpath(url, path):
     doc = lxml.html.fromstring(requests.get(url).text)
     return doc.xpath(path)
 
 
+def convert_pdf(filename, type='xml'):
```

Comment:
This can be imported from https://github.com/opencivicdata/pupa/blob/master/pupa/utils/generic.py#L64.